static/app/views/alerts/rules/issue/setupAlertIntegrationButton.tsx

```diff
@@ -54,6 +59,32 @@ export default class SetupAlertIntegrationButton extends DeprecatedAsyncComponen
       return null;
     }
 
+    if (organization.features.includes('messaging-integration-onboarding')) {
```

Comment:
should we only show this if somebody doesn't have a messaging integration installed?

Reply:
We should also check for plan right?

Reply:
@cathteng yup, `SetupAlertIntegrationButton` returns null if messaging integration is already installed:
https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/rules/issue/setupAlertIntegrationButton.tsx#L53-L55